### PR TITLE
Deprecated voice normalization rules

### PR DESCRIPTION
**Pull request by JensTrier**
These 3 PS cmdlet will be removed from Teams PS Module in the middle of February 2022. For now just adding the note about the deprecation. After the removal from TPM is done, I'll update these 3 pages to remove references to Skype for Business Online/Microsoft Teams.

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsVoiceNormalizationRule.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ Returns information about the voice normalization rules used in your organizatio
 Voice normalization rules convert telephone dialing requirements (for example, dialing 9 to access an outside line) to the E.164 phone number format used by Skype for Business Server.
 This cmdlet was introduced in Lync Server 2010.
 
+> [!NOTE]
+> This cmdlet will be deprecated from Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsVoiceNormalizationRule.md:21]:
```suggestion
**Note**: This cmdlet will be deprecated from Teams PowerShell Module.
```

File: skype/skype-ps/skype/Remove-CsVoiceNormalizationRule.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ Removes a voice normalization rule.
 Voice normalization rules are used to convert telephone dialing requirements (for example, dialing 9 to access an outside line) to the E.164 phone number format used by Skype for Business Server.
 This cmdlet was introduced in Lync Server 2010.
 
+> [!NOTE]
+> This cmdlet will be deprecated from Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Remove-CsVoiceNormalizationRule.md:21]:
```suggestion
**Note**: This cmdlet will be deprecated from Teams PowerShell Module.
```

File: skype/skype-ps/skype/Set-CsVoiceNormalizationRule.md [hunk 3]
```diff
@@ -17,6 +17,8 @@ Modifies a voice normalization rule.
 Voice normalization rules are used to convert a telephone dialing requirement (for example, dialing 9 to access an outside line) to the E.164 phone number format used by Skype for Business Server.
 This cmdlet was introduced in Lync Server 2010.
 
+> [!NOTE]
+> This cmdlet will be deprecated from Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsVoiceNormalizationRule.md:21]:
```suggestion
**Note**: This cmdlet will be deprecated from Teams PowerShell Module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**